Source/Android/jni/MainAndroid.cpp

```diff
@@ -515,7 +515,7 @@ Java_org_dolphinemu_dolphinemu_NativeLibrary_UpdateGCAdapterScanThread(JNIEnv*,
 
 JNIEXPORT void JNICALL Java_org_dolphinemu_dolphinemu_NativeLibrary_Initialize(JNIEnv*, jclass)
 {
-  // InitControllers ends up calling config code, and some config callbacks use RunAsCPUThread
+  // InitControllers ends up calling config code, and some config callbacks run as the CPU thread
```

Comment:
Removed at your suggestion.

Reply:
I'm fine with the HostThreadLock staying. It's technically not needed right now, but the performance penalty is basically non-existent since this code only runs once and the mutex shouldn't be locked when it runs, and some of the code being called is exactly the kind of code that might make assumptions about the caller being the host thread. So keeping the HostThreadLock makes sense for future proofing.